src/hotspot/cpu/x86/c1_LIRAssembler_x86.cpp

```diff
@@ -2945,12 +2945,17 @@ void LIR_Assembler::shift_op(LIR_Code code, LIR_Opr left, LIR_Opr count, LIR_Opr
   // So for this case, we simply swap register data around, so that the count value
   // can be put in ECX correctly.
   // TODO: A better solution would be modifying and fixing the register allocator.
```

Comment:
Probably put the `// FIXME` comment inside the `#ifdef`

Reply:
Done